operators/video_streaming/streaming_client_enhanced/python/tests/test_streaming_client_op_bindings.py

```
@@ -0,0 +1,255 @@
+    def test_streaming_mode_parameters(self, operator_factory, receive_frames, send_frames):
+        """Test operator creation with different streaming modes."""
+        op = operator_factory(receive_frames=receive_frames, send_frames=send_frames)
+        assert op is not None
+
+    def test_frame_validation_parameter(self, operator_factory):
+        """Test min_non_zero_bytes parameter."""
+        for threshold in [0, 100, 1000, 10000]:
+            op = operator_factory(min_non_zero_bytes=threshold)
+            assert op is not None
+
+    def test_operator_inheritance(self, streaming_client_op_class, holoscan_modules):
```

Ruff (0.14.2)

Unused method argument: `holoscan_modules`

(ARG002)